build.gradle

```diff
@@ -24,6 +24,8 @@ buildscript {
         classpath Deps.tools_kotlingradle
         classpath Deps.tools_appservicesgradle
     }
+
+    ext.espresso_version = '3.0.2'
```

Comment:
See comment above about version management.

Reply:
@rpappalax was there a reason you needed this version here or can it move to `Dependencies.kt` as well?